composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/familyGroupSettings/familyGroupMember/ModifyFamilyMemberScreen.kt

```diff
@@ -62,6 +65,30 @@ class ModifyFamilyMemberScreen(private val familyMember: FamilyMember) : Screen
         val familyGroupService = koinInject<IFamilyGroupService>()
         val familyGroupCredentialsRepository = koinInject<IFamilyGroupCredentialsRepository>()
 
+        var currentUserPermissionGroup by remember { mutableStateOf<FamilyGroupMemberPermissionGroup?>(null) }
+        var isLoading by remember { mutableStateOf(true) }
+        var isLastGuardian by remember { mutableStateOf(false) }
+        var allFamilyMembers by remember { mutableStateOf<List<FamilyMember>>(emptyList()) }
```

Comment:
Skoro `allFamilyMembers` jest używane tylko na starcie do określenia liczby guardianów, to pokusiłbym się o usunięcie go stąd i zadeklarowanie `val allFamilyMembers` bezpośrednio w `LaunchedEffect`.